2020/CG-04-28.md

```diff
@@ -30,6 +30,9 @@ Installation is required, see the calendar invite.
        Poll: Should wasm-c-api pursue safety, as discussed in this issue?
        Poll: If not, approve a new WebAssembly C API proposal at phase 0?
     1. Flexible vectors - [slides](https://github.com/penzn/wasm-long-vectors), [design/1332](https://github.com/WebAssembly/design/issues/1332). Poll: Approve phase 0 for flexible vectors.
+    1. Subclassing of WebAssembly classes. See https://github.com/WebAssembly/spec/issues/1107.
+       Discuss: What should we do with asynchronous compilation/instantiation APIs and subclassing?
+       Poll: Should we at least allow subclassing when used with a synchronous `new` instantiation?
```

Comment:
Note that this is already allowed: https://github.com/WebAssembly/spec/issues/1107#issuecomment-613490560.

Reply:
Thanks. I guess I'll close this PR for now.